packages/cli/src/workflows/workflow.service.ts

```diff
@@ -58,13 +60,33 @@ export class WorkflowService {
 		private readonly globalConfig: GlobalConfig,
 	) {}
 
-	async getMany(user: User, options?: ListQuery.Options, includeScopes?: boolean) {
+	async getMany(
```

Comment:
Maybe later we should split this into a endpoint that return workflows and folders and another endpoint that only return folders. That would require more refactoring to be able to reuse as much as possible.

Reply:
Also, we need to think wether we really need the `select` querystring parameter as this adds quite some complexity, and we most likely do not need it.